LadybugTools_Engine/Query/IsPollinationInstalled.cs

```diff
@@ -0,0 +1,78 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2023, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *
+ *
+ * The BHoM is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3.0 of the License, or
+ * (at your option) any later version.
+ *
+ * The BHoM is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+ */
+
+using BH.oM.Base.Attributes;
+using BH.oM.LadybugTools;
+using System.Linq;
+using System.Collections.Generic;
+using System.ComponentModel;
+using BH.oM.Python;
+using System.IO;
+using System;
+using System.Diagnostics;
+
+namespace BH.Engine.LadybugTools
+{
+    public static partial class Query
+    {
+        [Description("Return True if Pollination is installed to the currently supported version.")]
+        [Input("targetPollinationVersion", "The target Pollination build BHoM currently supports.")]
+        [Input("includeBuildNumber", "If true, the build number will be included in the comparison.")]
+        [Output("bool", "True if Pollination is installed to the currently supported version.")]
+        public static bool IsPollinationInstalled(string targetPollinationVersion, bool includeBuildNumber)
```

Comment:
I would put the defaults on this so we can update them once in the code for the majority of cases whenever we need to change the Pollination version rather than needing to update calls wherever it is made.